l5kit/l5kit/geometry/transform.py

```diff
@@ -70,6 +70,39 @@ def flip_y_axis(tm: np.ndarray, y_dim_size: int) -> np.ndarray:
     return tm
 
 
+def transform_points_batch(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarray:
+    """Transform a batch of sequence of points using a batch of transformation matrices.
+    Each sequence is transformed using its own matrix
+    Note this function assumes points.shape[-1] == matrix.shape[11] - 1, which means that last
```

Comment:
`points.shape[-1] == matrix.shape[-1] - 1` not `points.shape[-1] == matrix.shape[11] - 1`